@xen-orchestra/lite/src/components/TitleBar.vue

```diff
@@ -1,8 +1,10 @@
 <template>
```

Comment:
I don't think we need to change the structure to add a container here (unless there's a reason I can't see).

I think the only needed change is to add this to `style`:

```
.actions {
  margin-left: auto;
}
```